Gems/Atom/Feature/Common/Code/Source/Decals/DecalTextureArrayFeatureProcessor.cpp

```diff
@@ -549,16 +549,19 @@ namespace AZ
         {
             const auto materialAsset = QueueMaterialAssetLoad(materialId);
 
-            m_materialLoadTracker.TrackAssetLoad(handle, materialAsset);
-
             if (materialAsset.IsLoading())
             {
+                m_materialLoadTracker.TrackAssetLoad(handle, materialAsset);
```

Comment:
Maybe add a quick comment why this was moved here.

Reply:
I don't think a comment in the code is necessary since anyone reading it won't know about the history. But basically, before it was tracking loads for assets that were already loaded, which isn't necessary.